### ISSUE TITLE
E-Mails from Hydra are useless

### ISSUE THREAD

**Issue by jtojnar** [NixOS/hydra#527]
TL;DR: The messages arrive too late and duplicated (https://github.com/NixOS/hydra/issues/48).

Since I became GNOME maintainer, I started receiving e-mails from Hydra, for example, today I received this:

> The status of Hydra job ‘nixpkgs:trunk:gnome3.something.aarch64-linux’ has changed from "Success" to "Dependency failed".  For details, see

In four copies:

- https://hydra.nixos.org/build/66535829
- https://hydra.nixos.org/build/65999876
- https://hydra.nixos.org/build/66000927
- https://hydra.nixos.org/build/66000428

Few days ago my mailbox was overflowing with similar failures.

That would be fine if the messages arrived on time but they are three weeks late and the problem in question already resolved.

Additionally, the notification often concern themselves with a single dependency failure, unnecessarily flooding my inbox.

**Comment by jtojnar** [comment]
The mail headers showing that this is probably not caused on the way:

```
Delivered-To: jtojnar@gmail.com
Received: by 10.200.24.186 with SMTP id s55csp3240349qtj;
        Mon, 8 Jan 2018 21:48:27 -0800 (PST)
X-Google-Smtp-Source: ACJfBovsBVg8r5R2+L0/Nk2SWuBOy3ZbU8ybpdgZC/5R32jIEjkmyxv/joajX8XW06CFxeumHZLw
X-Received: by 10.223.144.1 with SMTP id h1mr13149194wrh.147.1515476907136;
        Mon, 08 Jan 2018 21:48:27 -0800 (PST)
ARC-Seal: i=1; a=rsa-sha256; t=1515476907; cv=none;
        d=google.com; s=arc-20160816;
        b=mgkzvoty6uMbkdcBhokxNXQT6pLuWo1H1O+Gz4TGDzPRgZL6P790ORGv2R4yXI27MR
         mn+By3Ny4tUM16UJYmgzNxSUwUyn5pPSw3DhjkvYu54C7GSaIcS1tcp2eMnkuq2Qul5O
         dwuPTFs9Od5vUY+h3rIe5fl5jIQ2EmySSBPuvKuLheSQ1uZhmRbCOlo9tlTNUv1QuAF/
         bKrO8f9kBHQ3lx3ckl2XEoe8GcheKdB+bAIy9jpZgD/1VthEKOfuOYTeSlEgXk0Btrz0
         BEezytnrfPIMrW07lyGjCX/5FvX7DqJqgXmiZBRgDfqUJQvEENzdb1ga5xKF9zY+439k
         2btg==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=message-id:content-transfer-encoding:mime-version:date:subject:from
         :to:arc-authentication-results;
        bh=rOODXIu3JkGWN6fM3+dgX84t/l/c3JsR3L5SXtfp7bI=;
        b=O+mFRNGAuDMv+fYcQLoUaAuyJwMqcQlB8FBDe59kVDLD2CAxnQdPHu3NrTwScBEnh5
         bX5pwXq+HWnAwWFzVhPqhORu6yOb6jtmGuUh6m1gx8ggqnNHXfEjb5f8enXb75mAFkoX
         pM3lWiYkeFQejGtQuz014ljBpqd7ZhH2jecz51P3o4PIdvxcDCI63iOqsmI4XT1ujIfD
         Z67IPU/H5kPDEOvMDKTOJZrCpHp9jmp10rGTg29os1vVegCqYCIUJ4fMaZX4GblcQUgQ
         27iFHzpF2mTxJRc2H8S+eYpU9aqXJ16DA8YdWH3slTi3wKqBoRsuoySuRtlzDO+moRja
         s5nQ==
ARC-Authentication-Results: i=1; mx.google.com;
       spf=softfail (google.com: domain of transitioning edolstra@gmail.com does not designate 46.4.67.10 as permitted sender) smtp.mailfrom=edolstra@gmail.com;
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=gmail.com
Return-Path: <edolstra@gmail.com>
Received: from hydra.nixos.org (static.10.67.4.46.clients.your-server.de. [46.4.67.10])
        by mx.google.com with ESMTP id l38si10174088wre.522.2018.01.08.21.48.26
        for <jtojnar@gmail.com>;
        Mon, 08 Jan 2018 21:48:27 -0800 (PST)
Received-SPF: softfail (google.com: domain of transitioning edolstra@gmail.com does not designate 46.4.67.10 as permitted sender) client-ip=46.4.67.10;
Authentication-Results: mx.google.com;
       spf=softfail (google.com: domain of transitioning edolstra@gmail.com does not designate 46.4.67.10 as permitted sender) smtp.mailfrom=edolstra@gmail.com;
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=gmail.com
Received: from localhost.localdomain (localhost [IPv6:::1]) by hydra.nixos.org (Postfix) with ESMTP id DA70A1C004F for <jtojnar@gmail.com>; Tue,
  9 Jan 2018 06:48:26 +0100 (CET)
To: Jan Tojnar <jtojnar@gmail.com>
From: Hydra Build Daemon <edolstra@gmail.com>
Subject: Dependency failed: Hydra job nixpkgs:trunk:gnome3.gnome-software.x86_64-linux
X-Hydra-Instance: https://hydra.nixos.org
X-Hydra-Project: nixpkgs
X-Hydra-Jobset: trunk
X-Hydra-Job: gnome3.gnome-software.x86_64-linux
X-Hydra-System: x86_64-linux
Date: Tue, 9 Jan 2018 06:48:26 +0100
MIME-Version: 1.0
Content-Type: text/plain; charset="UTF-8"
Content-Transfer-Encoding: quoted-printable
Message-Id: <20180109054826.DA70A1C004F@hydra.nixos.org>
```
